clients/vscode-hlasmplugin/src/test/suite/configurationsHandler.test.ts

```diff
@@ -32,7 +32,7 @@ suite('Configurations Handler Test Suite', () => {
     // 4 expressions - 2 for always recognize, 1 open code and 1 library file
```

Comment:
I've been considering just removing it... It verifies the correct number of regular expressions generated for highlighting autodetection (instead of actually verifying the regular expressions).